docs/topics/whatsnew-eap.md

```diff
@@ -20,664 +20,149 @@ All you need to do is to [change the Kotlin version](configure-build-for-eap.md)
-
-By default, this property is set to `true` to track the task dependency. To disable the use of the artifact in the compilation
-task, so that no task dependency is required, add the following in your `gradle.properties` file:
-
-```kotlin
-kotlin.build.archivesTaskOutputAsFriendModule=false
-```
-
-For more information, see the issue in [YouTrack](https://youtrack.jetbrains.com/issue/KT-69330).
-
-### Option to share JVM artifacts between projects as class files
-
-> This feature is [Experimental](components-stability.md#stability-levels-explained). 
-> It may be dropped or changed at any time. Use it only for evaluation purposes. 
-> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-61861/Gradle-Kotlin-compilations-depend-on-packed-artifacts). 
-> Opt-in is required (see details below).
->
-{type="warning"}
-
-In Kotlin %kotlinEapVersion%, we introduce a new approach that changes the way the outputs of Kotlin/JVM compilations, 
-such as JAR files, are shared between projects. With this approach, Gradle's `apiElements` configuration now has a secondary
-variant that provides access to the directory containing compiled `.class` files. When configured, your project uses this
-directory instead of requesting the compressed JAR artifact during compilation. This reduces the number of times JAR files
-are compressed and decompressed, especially for incremental builds.
-
-Our testing shows that this new approach can provide build performance improvements for Linux and macOS hosts. 
-However, on Windows hosts, we have seen a degradation in performance due to how Windows handles I/O operations when working with files.
-
-To try this new approach, add the following property to your `gradle.properties` file:
-
-```none
-kotlin.jvm.addClassesVariant=true
-```
-
-By default, this property is set to false and the `apiElements` variant in Gradle requests the compressed JAR artifact.
-
-> Gradle has a related property that you can use in your Java-only projects to only expose the compressed JAR artifact 
-> during compilation **instead** of the directories containing compiled `.class` files:
->
-> ```none
-> org.gradle.java.compile-classpath-packaging=true
-> ```
->
-> For more information on this property and its purpose, 
-> see Gradle's documentation on [Significant build performance drop on Windows for huge multi-projects](https://docs.gradle.org/current/userguide/java_library_plugin.html#sub:java_library_known_issues_windows_performance).
->
-{type="note"}
+### Bumped the minimum supported AGP version to 7.3.1
 
-We would appreciate your feedback on this new approach. Do you also see performance improvements? 
-Let us know by adding a comment in [YouTrack](https://youtrack.jetbrains.com/issue/KT-61861/Gradle-Kotlin-compilations-depend-on-packed-artifacts).
+Starting with Kotlin %kotlinEapVersion%, the minimum supported Android Gradle plugin version is 7.1.3.
```

Comment:
`7.1.3` here and `7.3.1` in the [line 134](https://github.com/JetBrains/kotlin-web-site/pull/4434/files#diff-7965e4f6aaaea3be50bb068bf47c0e2e56f67f0ea7d4caf2697303362874e72dR134)